src/gui/guiItemImage.h

```diff
@@ -0,0 +1,47 @@
+/*
+Minetest
+Copyright (C) 2013 celeron55, Perttu Ahola <celeron55@gmail.com>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#pragma once
+
+#include "irrlichttypes_extrabloated.h"
+#include "util/string.h"
+
+class Client;
+
+class GUIItemImage : public gui::IGUIElement
+{
+public:
+	GUIItemImage(gui::IGUIEnvironment *env, gui::IGUIElement *parent, s32 id,
+		const core::rect<s32> &rectangle, const std::string &item_name,
+		gui::IGUIFont *font, Client *client);
+
+	virtual void draw();
+
+	virtual void setText(const wchar_t *text)
+	{
+		m_label = core::stringw();
```

Comment:
`irr::core::string` seems a have a assignment operator for c-style strings so you should be able to just do
```suggestion
		m_label = text;
```

Reply:
Thanks!
(I wonder, why I did that. I thought, this was copied from somewhere, but no.)